Dockerfiles/README.md

```diff
@@ -41,7 +41,7 @@ docker start -ai reGen
 ```
 
 ### Stop
-To stop a running container (set it to maintenance mode first using a mangement site like [Artbot](https://tinybots.net/artbot/settings?panel=workers)):
```

Comment:
Please let me know that I understood this right, that this instruction is no longer necessary with the changed `docker stop` behavior.

Reply:
Yes, this was there to work around the non graceful exit.
I was wondering how that could be solved.